public/components/Status.react.js

```diff
@@ -21,46 +22,30 @@ export default class Status extends React.Component {
     }
 
     componentDidMount() {
-      if (!this.state.jobStatus || !this.state.jobStatus.length) {
-        this.fetchJobStatus();
-      }
+      this.fetchJobs();
+      this.jobCheck = setInterval(this.fetchJobs.bind(this), 15000);
     }
 
-    renderListItem(logItem) {
-
-      const itemTime = moment(logItem.started, 'x');
-      const rowClass = moment().subtract(1, 'days').isAfter(itemTime) ? 'row-warning' : '';
-
-      return (
-        <tr className={rowClass} key={logItem.id}>
-          <td>{itemTime.format('DD/MM/YYYY HH:mm:ss')}</td>
-          <td>{logItem.type}</td>
-          <td>{logItem.startedBy}</td>
-          <td>Currently Processing: {logItem.steps[0].type}<br />Remaining Steps: {logItem.steps.length}</td>
-        </tr>
-      );
+    componentWillUnmount() {
+      clearInterval(this.jobCheck);
     }
 
     render () {
-
       return (
-
         <div className="status">
-          <table className="status__table">
-            <thead className="status__header">
-              <tr>
-                <th>Started</th>
-                <th>Type</th>
-                <th>User</th>
-                <th>Progress</th>
-              </tr>
-            </thead>
-            <tbody className="status__results">
-              {this.state.jobStatus.sort((a, b) => a.id > b.id ? 1 : -1).map(this.renderListItem, this)}
-            </tbody>
-          </table>
+          <JobTable jobs={this.state.jobStatus} triggerRefresh={this.fetchJobs.bind(this)}  disableDelete={!this.props.config.permissions.tag_super_admin}/>
```

Comment:
Again here, super admin is going today so might be easier to deal with it before merge (especially since it's javascript you'll just end up with it being undefined :cry: )